src/main/java/com/scrive/pdftools/tools/utils/FileInput.java

```diff
@@ -0,0 +1,33 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.itextpdf.text.pdf.codec.Base64;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+public class FileInput {
+
+    static public byte[] getFileContent(JSONObject obj) throws IOException {
+        String localFilePath = obj.optString("localFilePath");
+        String base64Content = obj.optString("base64Content");
+        if (isValid(base64Content)) {
+            return Base64.decode(base64Content);
+        } else if (isValid(localFilePath)) {
+            try (RandomAccessFile f = new RandomAccessFile(localFilePath, "r")) {
+                return f.toString().getBytes();
+            }
+        } else {
+            throw new JSONException("Not valid file input from JSON");
+        }
+    }
+
+    static private boolean isValid(String string) {
```

Comment:
can be replaced by https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence- :)

Reply:
I replaced it. Just added a StringUtils.isEmpty(String) check in the if statements